src/mocsolver.jl

```diff
@@ -205,8 +254,44 @@ function compute_q!(sol::MoCSolution{T}, prob::MoCProblem) where {T}
             #     1 / keff * χ[g] * νΣf[g′] * φ[@region_index(i, g′)]
             #     for g′ in 1:NGroups
             # )
+            qig /= 4π
+            q[ig] = qig
```

Comment:
this seems to imply that I had a bug in the past, which makes me wonder why I got good benchmark results. Did I have a bug?